homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
Let's split `test_reauth` into 2 tests so we can test that the incorrect password is also able to recover

Reply:
Noted in 6f4b51d1eebeb8405feeff141d01bde7000425b1